components/Common/AvatarGroup/avatar/index.module.css

```diff
@@ -0,0 +1,3 @@
+.avatar {
+  @apply -ml-2 rounded-full border-2 border-white bg-neutral-100 first:ml-0 dark:border-neutral-950 dark:bg-neutral-800;
```

Comment:
Please read the collaborator guidelines of how to style components carefully.

Reply:
I had re-read normally it's good !